tests/TargetTest.php

```diff
@@ -177,16 +177,19 @@ public function testFormatMessage()
 
 class TestTarget extends Target
 {
-    public $exportInterval = 1;
+    public function __construct()
+    {
+        $this->setExportInterval(1);
+    }
```

Comment:
This is what we're forced to do since $exportInterval is now private.
Is that okay?

Reply:
Yes.